src/conda_actions.ts

```diff
@@ -284,12 +284,23 @@ const install_python = async (config: ConfigObject): Promise<void> => {
         '__setup_conda',
         `python=${config.python_version}`,
       ])
+    } else if (python_version.match(/^pypy((3\.\d+)?(=(\d+)?(\.\d+)?(\.\d+)?)?)?$/) !== null) {
+      await exec.exec('conda', [
```

Comment:
Please factor this out in a stand alone function so that it takes dependency as an argument.
Usage example
```js
await createCondaEnv(`python=${config.python_version}`)
```